### DISCUSSION TITLE
Install error with OS X 10.15

### DISCUSSION THREAD

**Discussion by symlyd** [Homebrew/discussions#374] [Getting started]
Hi, 

I am trying to install Homebrew so I can install QMK. 
I have run the installer on Terminal and get this error:
`error: RPC failed; curl 18 transfer closed with outstanding read data remaining

fatal: the remote end hung up unexpectedly

fatal: early EOF

fatal: index-pack failed

Error: Failure while executing; `git clone https://github.com/Homebrew/homebrew-core /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core` exited with 128.

Error: Failure while executing; `/usr/local/bin/brew tap homebrew/core` exited with 1.

Failed during: /usr/local/bin/brew update --force`

Anyone know what the exact issue is? 
I am on 10.15.6 to be specific.

**Comment by gromgit** [comment]
As the error message says:
```
fatal: the remote end hung up unexpectedly
```
See https://github.com/Homebrew/discussions/discussions/145 for some possible issues.

**Reply by symlyd** [reply]
Thanks I'll have a look and see if I can fix it.
